app/views/instances/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<% game = Game.find(@instance.game_id)%>
+<% player = User.find@instance.user_id%>
+
+<h2>Host: <%= player.nickname %></h2>
+<h2>Game: <%= game.name %></h2>
```

Comment:
As per previous comment, this will just then be `@game.nickname` and `@player.nickname`.

Reply:
Fixed, thank you for your help on this one!!